src/app/scripts/cac/map/cac-map-overlays.js

```diff
@@ -33,9 +31,14 @@ CAC.Map.OverlaysControl = (function ($, L) {
         return bikeShareFeatureGroup;
     }
 
-    // TODO: Implement
-    function bikeParkingOverlay() {
-        return L.featureGroup([]);
+    function bikeRoutesOverlay() {
+        // https://cac-tripplanner.cartodb.com/viz/501dbdc8-f4ea-11e4-8c9e-0e018d66dc29/public_map
```

Comment:
That's the link to the visualization itself.  I thought it might be helpful to document it, since the viz ID isn't in the tile URL itself.